libsrc/telestrat/gotoxy.s

```diff
@@ -19,8 +18,43 @@ gotoxy: jsr     popa            ; Get Y
 ; This function moves only the display cursor; it does not move the prompt position.
 ; In telemon, there is a position for the prompt, and another for the cursor.
 
-        sta     SCRY
-        jsr     popa
-        sta     SCRX
-        rts
+    sta     SCRY
+    
+
+    jsr     popa
+    sta     SCRX
+
+    jmp     update_adscr          ; Update adress video ram position when SCRY is modified 
```

Comment:
That line jumps to the next instruction.  You should comment it out.
And, add a comment on a line below it that says something like,
```asm
; Fall through
```

Reply:
Fixed